src/clj/rems/css/styles.clj

```diff
@@ -210,6 +210,9 @@
     (seq? obj) (map remove-nil-vals obj)
     :else obj))
 
+(defn- make-important [style]
+  (into {} (mapv (fn [[k v]] (vector k (str v " !important"))) style)))
```

Comment:
see also `medley.core/map-vals`

Reply:
Done. I was looking for something like this but didn't find it easily enough (even though it's used in the code four lines above...)